relprocotron/templates/release_activities.json.j2

```diff
@@ -0,0 +1,216 @@
+{
+  "release": {
+    "name": "{{ release_name }}",
+    "tag": "{{ release_tag }}",
+    "type": "{{ release_type }}",
+    "date": "{{ release_date }}",
+    "project_url": "{{ project_url }}",
+    "software_name": "{{ software_name }}",
+    "software_version": "{{ software_version }}"{% if comments %},
```

Comment:
Use easier-to-read styles for if/then statements in Jinja, more like the K&R style.